src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/VariantRecalibrator.java

```diff
@@ -487,6 +486,25 @@ else if (null != sequenceDictionary) {
         }
     }
 
+
+    private void orderAndValidateAnnotations(final GATKReportTable annotationTable){
+        annotationOrder = new ArrayList<Integer>(dataManager.annotationKeys.size());
+
+        for (int i = 0; i < annotationTable.getNumRows(); i++){
+            String serialAnno = (String)annotationTable.get(i, "Annotation");
+            for (int j = 0; j < dataManager.annotationKeys.size(); j++) {
+                if (serialAnno.equals( dataManager.annotationKeys.get(j))){
```

Comment:
Can you do a hash lookup against the `dataManager` to avoid an n^2 algorithm? (might not matter if these are small)

Reply:
Yeah I was being lazy, but I've never seen us use more than 7 annotations.